config/robots.rb

```diff
@@ -1483,62 +1483,47 @@
                    label: 'Bed 9'
                  },
                  bed(4).barcode => {
-                   purpose: 'LHR RT',
-                   states: ['passed'],
+                   purpose: 'LHR PCR 1',
+                   states: ['pending'],
                    label: 'Bed 4',
-                   display_purpose: 'LHR PCR 1',
-                   override_class: 'Robots::Bed::Heron',
-                   expected_plate_barcode_suffix: 'PP1'
```

Comment:
Consider reversing the changes we made for `override_class` in app/models/robots/robot.rb and app/models/robots/bed/base.rb ?

And also for `display_purpose` in app/models/robots/bed/base.rb